index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Portfolio</title>
+  <link href="https://fonts.googleapis.com/css?family=Arvo" rel="stylesheet" type="text/css">
+  <link href="styles/style.css" type="text/css" rel="stylesheet"/>
+</head>
+<div id="page">
+  <header class="page_header">
+    <h1>Joe McClenahan</h1>
+    <ul>
+      <li id="blogLink">Blog</li>
+      <li id="socialLink">Social</li>
+      <li id="portfolioLink">Portfolio</li>
+    </ul>
+  </header>
+  <div id="content">
+    <section id="blog">
+      <article class="blogTemplate" data-category="category">
+        <h1 class="blogTitle">Title</h1>
+        <h3 class="blogAuthor">Author</h3>
+        <time pubdate datetime="2000-01-01">Publish Time</time>
+        <p class="blogText">About Me</p>
+      </article>
+    </section>
+    <section class="social">
+      <ul id="social">
+        <li class="socialTemplate"><a href="">Social</a></li>
+      </ul>
+
+    </section>
+
+  </div>
+  <footer>
+  </footer>
+</div>
```

Comment:
Once you rename your body element.  Move your scripts up inside the body.  They should be the last thing in the body.  After the footer and before the closing body tag.